deploy/charts/harvester/values.yaml

```diff
@@ -335,6 +335,11 @@ service:
     ##
     type: ClusterIP
 
+    ## Specify the nodePort of HTTPs endpoint.
+    ## defaults to "0".
+    ##
+    httpsNodePort: 0
```

Comment:
why not set to the default value of 30443?

Reply:
iso mode: set httpsNodePort to 30443
app mode: default 0 to avoild port conflict in user's cluster